Watch Extension/ExtensionDelegate.swift

```diff
@@ -0,0 +1,58 @@
+import WatchConnectivity
+import WatchKit
+
+final class ExtensionDelegate: NSObject, WKExtensionDelegate {
+    func applicationDidFinishLaunching() {
+        // Perform any final initialization of your application.
+        if WCSession.isSupported() {
+            let session = WCSession.default
+            session.delegate = self
+            session.activate()
+        }
+    }
+
+    func applicationDidBecomeActive() {
+        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
+    }
+
+    func applicationWillResignActive() {
+        // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/jasonzurita/BabyPatterns/blob/9a1d22d8ad71ad19efd2b877aa3b663fe9f8e3a5/Watch Extension/ExtensionDelegate.swift#L19"></span>Line should be 120 characters or less: currently 285 characters<br />
<code>line_length</code> <code>ExtensionDelegate.swift:19</code></td>
    </tr>
  </tbody>
</table>